src/query.pegjs

```diff
@@ -2,6 +2,7 @@
   //TODO: disabled presets (e.g. mongo 2.2/2.6/3.0)
   //TODO: determine dependencies automatically
   if (!Array.isArray(options.disabledOperators)) options.disabledOperators = [];
+  if (options.caseInsensitiveOperators === undefined) options.caseInsensitiveOperators = false;
```

Comment:
Probably is better to check for existence with typeof options.caseInsensitiveOperators === 'undefined' or just options.caseInsensitiveOperators. undefined itself can be defined.